### ISSUE TITLE
speak() method not working on iOS

### ISSUE THREAD

**Issue by EvilBeer** [phonegap/phonegap-mobile-accessibility#28]
The "MobileAccessibility.speak()" method does not appear to be doing anything. The actual method call does take place, but it does not seem to fire anything on the iOS side. Works fine on Android.

**Comment by majornista** [comment]
I can't seem to replicate this in my test application. Could you provide any additional context? Is VoiceOver running on the iOS device. With VoiceOver running on the device, when debugging in Safari, you should be able to enter MobileAccessibility.speak("Hello world") in the console and hear "Hello world" spoken by VoiceOver on the device.

**Comment by EvilBeer** [comment]
The VoiceOver is indeed running. If I run MobileAccessibility.speak('Hello world') the method fires (as in, it's not breaking), but the VoiceOver does not react to that, so I'm assuming it's something to do with the bridge between Cordova and native, but currently that's just a guess.

**Comment by majornista** [comment]
Here's a link to the demo app that I'm using to test: https://github.com/majornista/phonegap-mobile-accessibility-test.

**Comment by merma90** [comment]
I have a similar problem. Is there any way to fix this? (running on android...)
